engine/class_modules/paladin/sc_paladin_holy.cpp

```diff
@@ -648,15 +648,35 @@ void paladin_t::generate_action_prio_list_holy_dps()
   cds->add_action( "use_items,if=(buff.avenging_wrath.up)" );
   cds->add_action( "seraphim" );
 
-  priority->add_action( this, "Shield of the Righteous" );
+  priority->add_action( "restore_mana,if=mana<10000", "Assume infinite mana." );
+  priority->add_action( this, "Shield of the Righteous",
+                        "if=buff.avenging_wrath.up|buff.holy_avenger.up|!talent.awakening.enabled",
```

Comment:
When I was testing on simpler APLs awakening was always a DPS downgrade if you were playing into it IE casting LoD or WoG. 
I'd like to see some sims (which I'll do later today if my build environment isn't destroyed) of the differences of playing into Awakening vs just hitting sotr

Reply:
Taking the awakening talent over Sanctified Wrath is a dps loss yes. However, if you are already running Awakening (i.e. every single situation besides doing 0 healing going for a parse in raid), it is worth it to fish for awakening up to 5 targets, unless wings or HA is up.

Reply:
Sorry I haven't been able to do sims work is pounding me but
from my understanding and from older sims even on single target hitting SoTR was more dps than fishing for awakening procs when you have the talent

Which the older sims had a worse APL but I'd still like to see the fact fishing for awakenings is more dps than just saying `heck this talent imma hit sotr on cd`

Reply:
I haven't had time to do sims but in the grand scheme of things if you think its a dps increase 
its a dps increase till proven other wise so i'll just merge it in